Assets/MixedRealityToolkit/_Core/Managers/MixedRealityManager.cs

```diff
@@ -309,124 +329,6 @@ private void OnDestroy()
 
         #endregion MonoBehaviour Implementation
 
-        private void InitializeAllManagers()
```

Comment:
Why move these and make them out of order?

Reply:
I moved them with the other Manager management code and put them in the order in which they are used there:

* Container Add
* Initialise
* ...

Reply:
Thinking the events we raise to the managers should be in their own region